projects/kit/components/progress/test/progress-bar.spec.ts

```diff
@@ -0,0 +1,114 @@
+import {ChangeDetectionStrategy, Component, Injectable, signal} from '@angular/core';
+import {type ComponentFixture, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {MutationObserverService} from '@ng-web-apis/mutation-observer';
+import {Subject} from 'rxjs';
+
+import {TuiProgress} from '../progress';
+
+describe('ProgressBar', () => {
```

Comment:
Let's rename spec title to `TuiProgressColorSegments` and file name to `projects/kit/components/progress/test/progress-color-segments.spec.ts`